.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,64 @@
+name: Cleanup
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Doing this every night is problematic. Could we do the cleanup on Sunday?
Scenario:
We have large builds/tests. Start a full build/test before shutting down the day to inspect the next morning.

Or maybe we can do cleanup for containers older than e.g. 2 days

Reply:
This is a plead for not doing nightly cleanups for any of our repos. At least wait one day so if you come back in the morning you can still inspect the failing tests

Reply:
The cleanup is before the nightly testing, so you can inspect the resources deployed by a given failing run for a full work day before it's cleaned up. And of course the logs in actions itself should have a reasonable amount of information to be useful too and that lasts much longer